src/navigator/types.tsx

```diff
@@ -56,7 +56,8 @@ export type StackParamList = {
   [Screens.AccountKeyEducation]:
     | undefined
     | {
-        nextScreen: keyof StackParamList
+        nextScreen?: keyof StackParamList
+        origin?: 'cabOnboarding' | 'onboarding' | 'settings'
```

Comment:
I think this can be derived instead 🤔

Reply:
I ended up using `origin?: 'cabOnboarding'` so this could probably be a boolean; however, it might be more extensible as is.